java/ql/src/Security/CWE/CWE-1104/MavenPomDependsOnBintray.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p><a href="https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/">Bintray and JCenter are shutting down on February 1st, 2022</a>.
+Relying upon repositories that are deprecated or slated to be shutdown can have unintended consequences;
```

Comment:
I'd recommend swapping out "slated" for something simpler for non-native speakers. Maybe:

```suggestion
Relying upon repositories that are deprecated or scheduled to be shutdown can have unintended consequences;
```